lib/browser/source.js

```diff
@@ -20,5 +108,9 @@ export function getSource(input) {
     return new FileSource(input);
   }
 
+  if (typeof input.read === "function") {
+    return new StreamSource(input, chunkSize);
+  }
+
   throw new Error("source object may only be an instance of File or Blob in this environment");
```

Comment:
Would you mind updating this error to mention that you can also put in a ReadableStream?